qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -68,6 +71,37 @@ public void testKNNIndexDefaultLegacyFieldMappingForceMerge() throws Exception {
         }
     }
 
+    // Ensure bwc works for binary force merge
+    public void testKNNIndexBinaryForceMerge() throws Exception {
+        // Binary is only supported on or after 2.16. Need to skip the others
+        if (getBWCVersion().isPresent() && Version.fromString(maybeRemoveSnapshotSuffix(getBWCVersion().get())).before(Version.V_2_16_0)) {
+            return;
+        }
```

Comment:
There is another way to do this skipping through brazil.gradle. wondering why we choose this way?

https://github.com/opensearch-project/neural-search/blob/main/qa/restart-upgrade/build.gradle#L134-L142

Reply:
fixed it